### ISSUE TITLE
bridged network not working with vagrant windows base box built with vagrant-windows and winRm

### ISSUE THREAD

**Issue by kamalim** [WinRb/vagrant-windows#22]
Hello:

I have built a windows 7 base box using vagrant-windows and winrm following all the steps that are mentioned in your README file.While this box works with nat and host only networking.It fails with bridged networking because winrm fails while enabling the network interfaces with the below error:

DEBUG winrm: WinRM Trying to connect
 INFO winrm: Execute: netsh interface ip set address "Local Area Connection 2" dhcp
DHCP is already enabled on this interface.

DEBUG winrm: 1
DEBUG winrm: netsh interface ip set address "Local Area Connection 2" dhcp EXIT STATUS 1
ERROR warden: Error occurred: translation missing: en.vagrant.errors.winrm_bad_exit_status
INFO interface: error: translation missing: en.vagrant.errors.winrm_bad_exit_status
translation missing: en.vagrant.errors.winrm_bad_exit_status

Looks like when the new adapter is added with bridged network being enabled windows VM automatocally detects the dhcp ip and so when winrm tries to enable it fails.

Any help will be really appreciable.

**Comment by pmorton** [comment]
@kamalim can you post your network configuration?

**Comment by kamalim** [comment]
config.vm.define :win1 do |win1_config|
        win1_config.vm.guest = :windows
        win1_config.vm.box = "win7_VL"
        win1_config.vm.boot_mode = :gui

```
    win1_config.vm.forward_port 3389, 3390, :name => "rdp", :auto => true
    win1_config.vm.forward_port 5985, 5985, :name => "winrm", :auto => true
    win1_config.vm.network :hostonly, "192.168.200.12"
```

It is working with the host only network but not with bridged

**Comment by pmorton** [comment]
@kamalim have you upgraded to 1.1 yet? If not I recommend upgrading. I just fixed #29 on the 1.1 compatible version of this plugin. To get the same behavior as above use:

``` ruby
config.vm.network :private_network, ip: "192.168.200.12"
```

**Comment by pmorton** [comment]
@kamalim If you would like me to look into this, please re-open.
